### ISSUE TITLE
CPAN bugs

### ISSUE THREAD

**Issue by epipping** [fc7/LaTeX-Decode#4]
Hi; there are a few open bugs here:

https://rt.cpan.org/Public/Dist/Display.html?Name=LaTeX-Decode

I'm particularly interested in

  https://rt.cpan.org/Public/Bug/Display.html?id=110678

which is a show-stopper with perl-5.26. Would you consider accepting a pull request for that and making a 0.05 release?

**Comment by epipping** [comment]
This is the kind of change I have in mind:
```diff
--- LaTeX/Decode.pm.bak	2017-07-04 19:06:07.000000000 +0200
+++ LaTeX/Decode.pm	2017-07-04 19:10:31.000000000 +0200
@@ -128,14 +128,14 @@
 
     if ( $scheme eq 'full' ) {
         $text =~ s/\\not\\($NEG_SYMB_RE)/$NEGATEDSYMBOLS{$1}/ge;
-        $text =~ s/\\textsuperscript{($SUPER_RE)}/$SUPERSCRIPTS{$1}/ge;
-        $text =~ s/\\textsuperscript{\\($SUPERCMD_RE)}/$CMDSUPERSCRIPTS{$1}/ge;
-        $text =~ s/\\dings{([2-9AF][0-9A-F])}/$DINGS{$1}/ge;
+        $text =~ s/\\textsuperscript\{($SUPER_RE)\}/$SUPERSCRIPTS{$1}/ge;
+        $text =~ s/\\textsuperscript\{\\($SUPERCMD_RE)\}/$CMDSUPERSCRIPTS{$1}/ge;
+        $text =~ s/\\dings\{([2-9AF][0-9A-F])\}/$DINGS{$1}/ge;
     }
 
     $text =~ s/(\\[a-zA-Z]+)\\(\s+)/$1\{\}$2/g;    # \foo\ bar -> \foo{} bar
     $text =~ s/([^{]\\\w)([;,.:%])/$1\{\}$2/g;     #} Aaaa\o, -> Aaaa\o{},
-    $text =~ s/(\\(?:$DIAC_RE_BASE|$ACCENTS_RE)){\\i}/$1\{i\}/g;
+    $text =~ s/(\\(?:$DIAC_RE_BASE|$ACCENTS_RE))\{\\i\}/$1\{i\}/g;
            # special cases such as '\={\i}' -> '\={i}' -> "i\x{304}"
 
     ## remove {} around macros that print one character
```

**Comment by fc7** [comment]
Hi Elias,
Sure, I will have a look at your PR this week-end and will release a new
version asap.
As I am very rarely using Perl these days, and I don't remember being
notified of those bugs from CPAN.
Thanks for contacting me directly via Github
François

2017-07-09 20:18 GMT+02:00 Elias Pipping <notifications@github.com>:

**Comment by epipping** [comment]
Cool, I've now opened #5 for this issue.

**Comment by fc7** [comment]
Included in CPAN release 0.05
